swift/ql/src/queries/Security/CWE-760/ConstantSalt.qhelp

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>Constant salts should not be used for password hashing. Data hashed using constant salts are vulnerable to dictionary attacks, enabling attackers to recover the original input.</p>
+  </overview>
+
+  <recommendation>
+    <p>Use randomly generated salts to securely hash input data.</p>
```

Comment:
Adding an hyphen because of the stacked modifiers.
```suggestion
    <p>Use randomly-generated salts to securely hash input data.</p>
```

Reply:
I thought we don't need a hyphen when the first part of a stacked modifier is an adverb (which is the case here), no?

Reply:
You are absolutely rigth 🤦🏻 
Please ignore this comment 😅